Makefile.am

```diff
@@ -73,7 +73,10 @@ bumpver:
 archive: po-pull
 	$(MAKE) distcheck
 
-local: po-empty
+local: all
+	[ -f ./po/en_US.po ] || \
+	msginit -i ./po/$(PACKAGE_NAME).pot -o ./po/en_US.po --no-translator || \
+	exit 1 ;
```

Comment:
I think you can drop the ``exit 1 ;`` line. If the *msginit* fails, *make* won't continue.